packages/math-inline/src/simple-question-block.jsx

```diff
@@ -106,7 +105,7 @@ const SimpleQuestionBlock = withStyles(theme => ({
     color: color.text(),
     background: color.background(),
     border: `1px solid ${color.primaryLight()}`,
-    width: '100%',
+    width: 'fit-content',
```

Comment:
Have you checked this on multiple screen sizes?

Reply:
yes, it looks good